### PR TITLE
Concur trigger doc

### PR DESCRIPTION
**Pull request by eeshansim**
### Which issues does this fix?
**DO NOT MERGE** - until Concur trigger PR (workato/workato#21885) is deployed

### PR submission checklist
- [x] Optimised every new image
- [x] Checked all content for grammatical errors (using Grammarly, for example)
- [x] Added new files to SUMMARY.md

### CODE REVIEW

File: connectors/concur/new-expense-report-trigger.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <td>When first started, this recipe should pick up events from</td>
+      <td>
+        Expense reports created or submitted after this time will be processed by the recipe.<br>
+        If no Approval status is selected or if <b>Not submitted</b> status is selected, this trigger will pick up expense reports when they are first created. Otherwise, it will pick up expense reports when they are submitted.
+      </td>
+    </tr>
+    <tr>
+      <td>Approval status</td>
+      <td>
+        Select an Approval status to filter from this list of statuses:<br>
```
Comment (eeliang) [inline connectors/concur/new-expense-report-trigger.md]: "to filter from this list of statuses" feels out of place.

Suggested: Filter the expense reports by selecting an Approval status from this list:

File: connectors/concur/new-expense-report-trigger.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        - Needs to be resubmitted<br>
+        - Approved<br>
+        - Expired in approval queue<br><br>
+        You can also provide a custom status. Refer to SAP Concur <a href="https://developer.concur.com/api-reference/expense/expense-report/v3.reports.html">API documentation</a> for more information about these statuses.
+      </td>
+    </tr>
+    <tr>
+      <td>Payment status</td>
+      <td>
+        Select a Payment status to filter from this list of statuses:<br>
```
Comment (eeliang) [inline connectors/concur/new-expense-report-trigger.md]: "Suggested: Filter the expense reports by selecting a Payment status from this list:"

File: connectors/concur/new-expense-report-trigger.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+title: Workato connectors - SAP Concur New expense report trigger
+date: 2018-10-04 06:00:00 Z
+---
+
+# SAP Concur - New expense report trigger
+
+## New expense report trigger
+
```
Comment (eeliang) [inline connectors/concur/new-expense-report-trigger.md]: Some introduction with polling:

This trigger picks up expense reports when they are created or submitted. Each report is processed as a separate job. It checks for new reports once every poll interval. The polling interval is determined by your Workato plan. Check the [Pricing and Plans page](https://www.workato.com/pricing?audience=general) to find out more.

**Comment by eeshansim** [comment]
Do not merge until PR is ready

**Pull request merged**